easypost/__init__.py

```diff
@@ -961,11 +961,10 @@ class Report(AllResource, CreateResource):
     def create(cls, api_key=None, **params):
         requestor = Requestor(api_key)
         url = "%s/%s" % (cls.class_url(), params['type'])
-        wrapped_params = {cls.class_name(): params}
-
-        response, api_key = requestor.request('post', url, wrapped_params, False)
+        response, api_key = requestor.request('post', url, params)
```

Comment:
When named-default parameters are present, it is a best practice to invoke the function with named arguments as well, if they are different from the default values.

The signature of the request method, per https://github.com/EasyPost/easypost-python/pull/91/files#diff-311814b232626ffa6fd7feb3ee0aa9a4L256, is:

`def request(self, method, url, params=None, apiKeyRequired=True):`

Would recommend invoking as:

`response, api_key = requestor.request('post', url, params=params)`

This changes from `apiKeyRequired` from `False` to True`

It's not clear why "unwrapping" the params is the right fix

Reply:
Also, CI is failing, so we should:

1. Whenever adding code/bug fix, add an accompanying test
2. Get CI to pass

Reply:
CI should hopefully pass if I rebased correctly. I've added back in the missing `False`. Kept params as is since it matches the rest of the library where params are also used.